### DISCUSSION TITLE
How to calculate maximum power flow in a branch?

### DISCUSSION THREAD

**Discussion by mdzalfirdausi** [e2nIEE/pandapower#2475] [Q&A] [answered]
I'm currently reproducing the OPF results from the following paper: [Adjustable Robust OPF With Renewable Energy Sources](https://ieeexplore.ieee.org/abstract/document/6575173) with pandapower. Since my background isn't EE, I'm striving to find how to calculate maximum power flow in a branch ( such in the constraint (3) ). The constraints are as follows:
![image](https://github.com/user-attachments/assets/4eb433d4-358f-45bc-8ed5-99778446b109)

After contacting the author of the paper, he sent me the dataset he used, but unfortunately for me, that's not very helpful since the dataset is only the original MATPOWER data, and still the formula to calculate the maximum power flow given the data from case14 pandapower is vague. Many thanks for any help.

**Comment by Fabsit** [comment, marked as answer]
this is a parameter

**Reply by mdzalfirdausi** [reply]
thanks for your reply. I'm wondering, how to convert that parameter's unit into MW, i.e. it is originally provided in MVA ([in MATPOWER manual](https://matpower.org/docs/MATPOWER-manual.pdf)).
